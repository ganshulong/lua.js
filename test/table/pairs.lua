local t = {
    1, 2, 3,
    foo=20, bar=40,
    [next] = 1
}

for k,v in pairs(t) do
    print(type(k), v)
end
