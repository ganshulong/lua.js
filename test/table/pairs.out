number 1
number 2
number 3
string 20
string 40
function 1
